### DISCUSSION TITLE
How do we set the refresh interval?

### DISCUSSION THREAD

**Discussion by drjasonharrison** [SigNoz/signoz#1289] [Q&A]
I can't seem to find any setting or documentation on setting the refresh interval for the Metrics, Traces or Dashboards.

This issue appears to track the issue of refresh controls: https://github.com/SigNoz/signoz/issues/183

**Comment by pranay01** [comment]
@drjasonharrison as of now,  auto refresh is not supported. If you are interested in this, would be great if you can add an upvote on the linked issue - so that we can take that input in prioritisation
